app/services/kis_trading_service.py

```
@@ -0,0 +1,367 @@
+) -> Dict[str, Any]:
+    """분석 결과를 기반으로 KIS 국내 주식 매수 주문 처리"""
+    from app.core.db import AsyncSessionLocal
+    from app.services.stock_info_service import StockAnalysisService
+
+    # 1. 예수금 확인
+    # KISClient.get_balance returns dict with 'output2' list usually containing balance info
+    # We need a simpler balance check. 
+    # For now, let's assume the caller might have checked, or we check here.
+    # Fetching balance is an API call.
+    
+    balance_info = await kis_client.get_balance()
```

Ruff (0.14.6)

Local variable `balance_info` is assigned to but never used

Remove assignment to unused variable `balance_info`

(F841)